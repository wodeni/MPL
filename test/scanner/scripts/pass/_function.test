float func (int a, int b) {return 1.0;}
