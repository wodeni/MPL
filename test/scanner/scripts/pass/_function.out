FLOAT
FUNC
LPAREN
INT
ID
COMMA
INT
ID
RPAREN
LBRACE
RETURN
FLOATLIT
SEMI
RBRACE
